Gemfile

```diff
@@ -15,6 +15,8 @@ gem "minitest-stub-const"
 gem "sd_notify"
 
 gem "jruby-openssl", :platform => "jruby"
+# needed for rubocop
+gem "racc", :platform => "jruby"
```

Comment:
Is this a bug in rubocop that we need to add it? (Or bundled or JRuby)

Reply:
Probably JRuby, but it's recent, and will probably be fixed in the future?  Log from Actions:
```
Run bundle exec rake rubocop
rake aborted!
LoadError: no such file to load -- racc/info
/home/runner/.rubies/jruby-9.2.15.0/bin/bundle:23:in `<main>'
```